### PR TITLE
[plugin.video.pcloud-video-streaming] 1.2.6

### PR DESCRIPTION
**Pull request by gdomenici**
Sorry, for some reason this pull request is weird... it tries to pull in commit 87c186b (corresponding to plugin version 1.2.4) and commit 1afcfd5 (corresponding to plugin version 1.2.5) although both these versions were already successfully merged as a result of previous pull requests. 

The only relevant commits are d1b5db7 and b2d1e67.

To be honest I am not sure how to fix it... help?

### Description
* Fixed issue #4, occurring with files whose internal pCloud ID ('fileID') was larger than a given threshold.
* Updated Python dependency to 2.19.0 to match Helix.
* Changed language files from .XML to .PO format.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by Razzeee** [comment]
Do you know how to rebase? If yes, just skip the commits you don't want.

**Comment by gdomenici** [comment]
Thanks @Razzeee . Good point. I have never done rebasing, but have read about it. Will give it a go and then submit another pull request.

**Comment by Razzeee** [comment]
@gdomenici 
by the way you can just (force) push to your branch and it will update this PR (which then needs to get reopend)

**Comment by gdomenici** [comment]
Reopening... new attempt

**Comment by gdomenici** [comment]
OK, apparently no more merge conflicts. Kindly pull if all OK.

Thanks!

Guido

**Comment by MartijnKaijser** [comment]
It has merge conflicts

File: plugin.video.pcloud-video-streaming/resources/language/English/strings.po [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+msgctxt "#30114"
+msgid "Delete from pCloud..."
+msgstr ""
+
+msgctxt "#30115"
+msgid "Confirm Delete"
+msgstr ""
+
+msgctxt "#30116"
+Delete '{0}' from pCloud?</string>
```
Comment (Rechi) [inline plugin.video.pcloud-video-streaming/resources/language/English/strings.po]: `Msgid "` is missing

**Comment by gdomenici** [comment]
Whoa @Rechi , eagle-eye. Fixed the issue with missing msgid in strings.po. Thanks.
@MartijnKaijser: you said 
> It has merge conflicts

But the GitHub UI tells me "This branch has no conflicts with the base branch". Where should I look then to figure out where those conflicts are?

**Comment by Rechi** [comment]
Don't know what @MartijnKaijser have seen, but there are no merge conflicts.

**Pull request merged**